calendar-tool/tool/src/java/org/sakaiproject/calendar/entityproviders/CalendarEventEntityProvider.java

```diff
@@ -112,21 +120,23 @@ public boolean entityExists(final String id) {
 		// API
 		Calendar cal;
 		try {
-			cal = calendarService.getCalendar(String.format(
-					"/calendar/calendar/%s/main", siteId));
+			//get all events from the merged calendars
+			CalendarEventVector calendarEventVector = calendarService.getEvents(calendarService.getCalendarReferences(siteId), getYearTimeRange(new CalendarUtil()));
```

Comment:
I wonder if including the merged calendars should be an option on the URL as if we push this change back to Sakai, someone else who is consuming the API will start getting events they might not have been expecting. WDYT?